.travis.yml

```diff
@@ -43,9 +43,9 @@ matrix:
         - os: linux
           compiler: clang-3.6
           env: CONFIG_OPTS="enable-asan"
-        - os: linux
-          compiler: clang-3.6
-          env: CONFIG_OPTS="enable-msan"
+#        - os: linux
+#          compiler: clang-3.6
+#          env: CONFIG_OPTS="enable-msan"
```

Comment:
"Just clang" is clang-3.4 which is too ancient for the sanitizer.

It's all already in the job logs, expand the "build system information" tab.